src/raftstore/store/config.rs

```diff
@@ -129,6 +133,8 @@ impl Default for Config {
             max_leader_missing_duration: Duration::from_secs(DEFAULT_MAX_LEADER_MISSING_SECS),
             snap_apply_batch_size: DEFAULT_SNAPSHOT_APPLY_BATCH_SIZE,
             lock_cf_compact_interval_secs: DEFAULT_LOCK_CF_COMPACT_INTERVAL_SECS,
+            check_stall_readonly_query_interval: DEFAULT_CHECK_STALL_RO_QUERY_INTERVAL_MS,
```

Comment:
It's better to keep two name consistent, the only difference should be the case.
s/DEFAULT_CHECK_STALL_RO_QUERY_INTERVAL_MS/DEFAULT_CHECK_STALL_READONLY_INTERVAL_MS/